frontend/src/app/jwt-interceptor.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { HttpInterceptor, HttpRequest, HttpEvent, HttpHandler } from '@angular/common/http';
+import { Observable } from 'rxjs/Observable';
+
+@Injectable()
+export class JwtInterceptorService implements HttpInterceptor {
+
+  token: string;
```

Comment:
`string | null` to be more accurate (if we ever activate `strictNullChecks`...)?